go/vt/vtgate/planner.go

```diff
@@ -43,10 +52,53 @@ func NewPlanner(vschema *vindexes.VSchema, cacheSize int) *Planner {
 	return plr
 }
 
+// LoadVSchema loads the VSchema from the topo. The function does
+// not return an error. It instead logs warnings on failure.
+func (plr *Planner) LoadVSchema() {
+	formal := &vindexes.VSchemaFormal{
+		Keyspaces: make(map[string]vindexes.KeyspaceFormal),
+	}
+	keyspaces, err := plr.serv.GetSrvKeyspaceNames(context.TODO(), plr.cell)
```

Comment:
This is also the function you'd call when notified of a change, which is why I didn't add the context. Do you plan to create one in such cases?